src/client.jl

```diff
@@ -321,7 +321,7 @@ response(::Type{Nothing}, resp::Downloads.Response, body) = nothing::Nothing
 response(::Type{T}, resp::Downloads.Response, body) where {T <: Real} = response(T, body)::T
 response(::Type{T}, resp::Downloads.Response, body) where {T <: String} = response(T, body)::T
 function response(::Type{T}, resp::Downloads.Response, body) where {T}
-    ctype = header(resp, "Content-Type", "application/json")
+    ctype = header(resp, "content-type", "application/json")
```

Comment:
Instead of this we should do a case insensitive compare in the `header` method. Line 315.

Reply:
Done